index.html

```diff
@@ -7,7 +7,10 @@
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+      <button id="on_hover">Tooltip on hover</button>
+      <button id="on_click">Tooltip on click</button>
```

Comment:
Good job. You can also create a button with modal window 🙂

Reply:
Oh, sorry, for me the demo looked like this and I didn't notice the third button 😅


<img src="https://github.com/coolassassin/tooltip-and-modal/assets/77266592/9824c9d2-fd21-4b82-b212-cf01281bfce1" height=350>